pom.xml

```diff
@@ -437,6 +438,42 @@ Licensed under the MIT license. See License.txt in the project root. -->
               </descriptors>
             </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.vafer</groupId>
+            <artifactId>jdeb</artifactId>
+            <version>1.4</version>
+            <executions>
+              <execution>
+                <phase>package</phase>
+                <goals>
+                  <goal>jdeb</goal>
+                </goals>
+                <configuration>
+                  <controlDir>${project.build.directory}</controlDir>
```

Comment:
P2: The documentation makes it look like there's already a templating mechanism built into jdeb, using the [[double square brackets]].  Can you confirm?